Scripts/DCS-BIOS/lib/modules/aircraft_modules/C-130J.lua

```diff
@@ -222,6 +222,15 @@ local devices = {
 -- Center Console
 
 -- Pilot Lighting Panel
+local PLT_LIGHTING_PANEL = "Pilot Lighting Panel"
+C_130J:define3PosTumb("PLT_MASTER_LIGHTING_MODE", devices.LIGHTING_PANELS, 3029, 1337, PLT_LIGHTING_PANEL, "Lighting Mode Master Switch")
```

Comment:
I won't comment on all the switches, but generally I try to include something to indicate the panel the switch is on in the switch id. This helps prevent confusion if there are perhaps multiple controls that would otherwise be named similarly (consider `PLT_PANEL_LIGHT_BRIGHTNESS` could also reasonably be the name of any knob that controls the brightness of any panel anywhere, for example).

I'd consider something like `PLT_LIGHTING_...` or similar as the prefix. Perhaps it's even worth including the location of the panel (I'm not sure if the overhead console also has a lighting panel), so something like `PLT_CC_LIGHTING_...` would also seem reasonable to me, especially since the large majority of the aircraft's controls are divided into panels on one of a few consoles.